modules/@apostrophecms/doc-type/index.js

```diff
@@ -71,6 +71,111 @@ module.exports = {
       }
     };
   },
+  commands(self) {
+    if (
+      self.__meta.name === '@apostrophecms/any-doc-type' ||
+      self.__meta.name === '@apostrophecms/global' ||
+      self.apos.instanceOf(self, '@apostrophecms/any-page-type') ||
+      self.apos.instanceOf(self, '@apostrophecms/page-type') ||
+      self.options.canCreate === false ||
+      self.options.showPermissions === false
+    ) {
+      return null;
+    }
+
+    return {
+      add: {
+        [`${self.__meta.name}:manager`]: {
```

Comment:
Definitely feeling how nice it would be to infer all this `self.__meta.name` stuff during the resolution when there is no `:`. But that can be added later.